scss/_functions.scss

```diff
@@ -48,6 +48,58 @@
   @return $string;
 }
 
+// map-sort-by-values
+// Sorts a map ascending by values
+// Used for grid-breakpoints and container-max-widths sorting
+//
+// @author Andrew Luca
```

Comment:
No need for the `@` comments here. You'll get the credits in the git history if this gets merged 😉

Reply:
Thanks a lot!